Chat App/Chat App/ViewController.swift

```diff
@@ -9,11 +9,69 @@ import UIKit
 
 class ViewController: UIViewController {
 
+    private let topChatView = ChatView()
+    private let separatorView = UIView()
+    private let bottomChatView = ChatView()
+    private let switchButton = DayLightSwitch(isOn: false)
+    
     override func viewDidLoad() {
         super.viewDidLoad()
-        // Do any additional setup after loading the view.
+        
+        view.addSubview(topChatView)
+        view.addSubview(separatorView)
+        view.addSubview(bottomChatView)
+        view.addSubview(switchButton)
+        separatorView.backgroundColor = Constants.ChatAppColors.separatorViewColor
+        setUpConstraintsForSubviews()
+        switchButton.delegate = self
     }
 
-
+    // MARK: layout constraints
+    private func setUpConstraintsForSubviews() {
+        topChatView.translatesAutoresizingMaskIntoConstraints = false
+        separatorView.translatesAutoresizingMaskIntoConstraints = false
+        bottomChatView.translatesAutoresizingMaskIntoConstraints = false
+        switchButton.translatesAutoresizingMaskIntoConstraints = false
```

Comment:
It is a good practice to place any repetitive assignment to a loop, like this:

`[topChatView, separatorView, bottomChatView, switchButton].forEach({`
`$0.translatesAutoresizingMaskIntoConstraints = false`
`})`